### ISSUE TITLE
User Library missing 

### ISSUE THREAD

**Issue by aim4pole** [lordscales91/Simple-POJO-Generator#2]
After lot of trial and error I reached this point

D:\Simple-POJO-Generator-master\bin>java -jar Simple-POJO-Generator.jar -gen xlsx -i "input.xlsx" -o tmp
Exception in thread "main" org.apache.poi.POIXMLException: java.lang.reflect.InvocationTargetException
        at org.apache.poi.xssf.usermodel.XSSFFactory.createDocumentPart(XSSFFactory.java:62)
        at org.apache.poi.POIXMLDocumentPart.read(POIXMLDocumentPart.java:456)
        at org.apache.poi.POIXMLDocument.load(POIXMLDocument.java:174)
        at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:278)
        at util.xls.PojoXlsGenerator.generate(PojoXlsGenerator.java:64)
        at util.xls.PojoXlsGenerator.main(PojoXlsGenerator.java:53)
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
        at java.lang.reflect.Constructor.newInstance(Unknown Source)
        at org.apache.poi.xssf.usermodel.XSSFFactory.createDocumentPart(XSSFFactory.java:60)
        ... 5 more
Caused by: java.lang.NoSuchMethodError: org.apache.xmlbeans.XmlOptions.setLoadEntityBytesLimit(I)Lorg/apache/xmlbeans/XmlOptions;
        at org.apache.poi.POIXMLTypeLoader.<clinit>(POIXMLTypeLoader.java:50)
        at org.apache.poi.xssf.model.ThemesTable.<init>(ThemesTable.java:84)
        ... 10 more

**Comment by lordscales91** [comment]
I removed the library intentionally, libraries shouldn't be inside a repository. Probably I should setup Maven for this project but... I'm not really maintaining it, it was just a tool that I created for personal use but decided to share it just in case someone finds it useful, so I went for the easy way and set up an Eclipse User library. That exception is probably because you are using wrong POI versions. Download them from @chanllen [fork](https://github.com/chanllen/Simple-POJO-Generator/tree/master/lib) or simply use his version. 
I'm glad to see there are at least two people that found this tool useful, if I third one comes up maybe I decide to continue working on it haha~
